python/ray/autoscaler/v2/instance_manager/config.py

```diff
@@ -15,6 +17,15 @@ class Provider(Enum):
     LOCAL = 6
 
 
+class ConfigReader(ABC):
+    """An interface for reading instance configs."""
```

Comment:
instance config and cluster config are the same thing right? We should pick and stick with one name for consistency.